org-formation/organization.yaml

```diff
@@ -287,6 +288,19 @@ Organization:
         budget-alarm-threshold: 20000
         budget-alarm-threshold-email-recipient: aws-nlpsandbox@sagebase.org
 
+  IncludeAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: IncludeAccount
+      RootEmail: aws-include-prod@sagebase.org
+      Alias: org-sagebase-include
+      Tags:
```

Comment:
@zaro0508, is there a tag we can add to enable all the resource that are created in this account to be tagged with a specific ProgramCode?

Reply:
Not that i'm aware of.  This tag is for the account and I believe we will be separating cost by account.  Which essentially means that we will be applying one cost center to the entire account.

Now that I think about it we should add a `CostCenter` tag to every account we create.  Please do that.